core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.ts

```diff
@@ -68,6 +69,9 @@ export class OpportunitiesListItemComponent {
   cardsAvailable: number = 0;
   onMobile!: boolean;
   resizeSubscription!: Subscription;
+  translationLengthLabel: string = '';
+  translationLengthLabelRequired: boolean = false;
+  translationLengthInWords: number = 0;
```

Comment:
I don't know if these variables are necessary. Seems simple enough to do everything in the html.

Reply:
Well I just followed the same pattern we had for other labels. But since these variables are only the use case for a very particular part, it's better to move it to html, Thanks.